pkgs/by-name/pi/pint/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, makeWrapper
+, curl
+, perl
+, git
+, nix-update-script
+, testers
+, pint
+}:
+
+buildGoModule rec {
+  pname = "pint";
+  version = "0.51.0";
+
+  src = fetchFromGitHub {
+    owner = "cloudflare";
+    repo = "pint";
+    rev = "v${version}";
+    hash = "sha256-d3KZPPeQJBqdrr81YLusYHc5jLChC1Rf5SYeP/QMeo8=";
+  };
+
+  vendorHash = "sha256-9tJL33Qtu0xAmRnAP0BwM6CIfGs+GEG864e89XrpfLM=";
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  subPackages = [ "cmd/pint" ];
+
+  CGO_ENABLED = 0;
+
+  ldflags = [
+    "-s"
+    "-w"
```

Comment:
```suggestion
    "-w"
    "-buildid="
```

Reply:
I understand this flag is best for reproducibility but it is a very opinionated choice and is not widely used in nixpkgs. If you don't mind I prefer not to add it as it is more a Go toolchain "problem", what do you think?

Reply:
I'm not sure why its very opinionated, I don't see a good reason to include the buildid. But you're right that this should be part of the Go builder. I'd say the same applies for -s -w.